easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.6-foss-2016b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'MakeCp'
+
+name = "CD-HIT"
+version = "4.6.6"
+
+homepage = 'http://weizhong-lab.ucsd.edu/cd-hit/'
+description = """ CD-HIT is a very widely used program for clustering and 
+ comparing protein or nucleotide sequences."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/weizhongli/cdhit/archive/']
+sources = ['V%(version)s.tar.gz']
```

Comment:
@boegel They are not 100 % the same but pretty close, only the case of the makefile is different:

```
> LANG=en diff cdhit-4.6.6/ cd-hit-v4.6.6-2016-0711/
Only in cdhit-4.6.6/: Makefile
Common subdirectories: cdhit-4.6.6/cd-hit-auxtools and cd-hit-v4.6.6-2016-0711/cd-hit-auxtools
Common subdirectories: cdhit-4.6.6/doc and cd-hit-v4.6.6-2016-0711/doc
Only in cd-hit-v4.6.6-2016-0711/: makefile
Common subdirectories: cdhit-4.6.6/psi-cd-hit and cd-hit-v4.6.6-2016-0711/psi-cd-hit
```